src/seedu/addressbook/AddressBook.java

```diff
@@ -426,9 +427,21 @@ private static String executeAddPerson(String commandArgs) {
 
         // add the person as specified
         final String[] personToAdd = decodeResult.get();
+        return upperCasePersonToAdd(personToAdd);
+    }
+
+    /**
+     * Convert each person's name to upper case for all persons to be added to the address book. 
+     * @param personToAdd
+     * @return feedback display message for the operation result with upper case name
+     */
+	private static String upperCasePersonToAdd(final String[] personToAdd) {
+		for (int i = 0; i < personToAdd.length; i++) {
+        	personToAdd[i] = personToAdd[i].toUpperCase();
+        }
```

Comment:
Good job for adding header comment. It should be `Converts` instead of `Convert`, as it reads like `This method converts ...`. You need to add description to `@param`.
Please fix the indentation and change the tab to spaces in Eclipse, see [here](http://stackoverflow.com/questions/407929/how-do-i-change-eclipse-to-use-spaces-instead-of-tabs)